src/RepoToolset/XUnit.targets

```diff
@@ -43,14 +43,14 @@
 
     <PropertyGroup Condition="'$(_RunOnCore)' == 'true'">
       <!-- TODO: xUnit doesn't support -html on CoreCLR currently (see https://github.com/xunit/xunit/issues/977) -->
-      <_TestRunnerCommand>"$(DotNetTool)" exec --depsfile "$(_CoreDepsPath)" --runtimeconfig "$(_CoreRuntimeConfigPath)" "$(NuGetPackageRoot)xunit.runner.console\$(XUnitVersion)\tools\netcoreapp1.0\xunit.console.dll" "$(TargetPath)" -noautoreporters -xml "$(_TestResultsXmlPath)"</_TestRunnerCommand>
+      <_TestRunnerCommand>"$(DotNetTool)" exec --depsfile "$(_CoreDepsPath)" --runtimeconfig "$(_CoreRuntimeConfigPath)" "$(NuGetPackageRoot)xunit.runner.console\$(XUnitVersion)\tools\netcoreapp1.0\xunit.console.dll" "$(TargetPath)" -noautoreporters -xml "$(_TestResultsXmlPath) $(_TestRunnerAdditionalParameters)"</_TestRunnerCommand>
```

Comment:
>_TestRunnerAdditionalParameters [](start = 281, length = 31)

I'd call this ```XUnitRunnerAdditionalArguments``` 
1) ```_``` prefixed properties are "private", this would be something repos should be setting, so it should not be private. 
2) The arguments are specific to XUnit. If we supported NUnit as well they would likely be different.

Reply:
fixed.